tools/ci_connector_ops/ci_connector_ops/pipelines/actions/environments.py

```diff
@@ -608,6 +608,9 @@ def with_integration_base_java_and_normalization(context: PipelineContext, build
         "git",
     ]
 
+    if context.connector.technical_name == "destination-snowflake":
```

Comment:
this should live in BASE_DESTINATION_NORMALIZATION_BUILD_CONFIGURATION, just putting it here for testing